sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusClientFactory.cs

```diff
@@ -62,8 +62,16 @@ private ServiceBusConnectionInformation ResolveConnectionInformation(string conn
             IConfigurationSection connectionSection = _configuration.GetWebJobsConnectionStringSectionServiceBus(connectionSetting);
             if (!connectionSection.Exists())
             {
+                // A common mistake is for developers to set their `connection` to a full connection string rather
+                // than an informational name.  If the value validates as a connection string, redact it to prevent
+                // leaking sensitive information.
+                if (IsServiceBusConnectionString(connectionSetting))
+                {
+                    connectionSetting = "<< REDACTED >> (a full connection string was incorrectly used instead of a connection name)";
```

Comment:
```suggestion
                    connectionSetting =  "<< REDACTED >> (a full connection string was incorrectly used instead of a connection setting name)";
```